install.sh

```diff
@@ -114,124 +193,303 @@ else
   fi
 fi
 
-rosdep -q update
 echo -e "\e[32mDone: Configuring rosdep\e[0m"
 echo ""
 
-
-echo -e "\e[94mConfiguring Clearpath Setup\e[0m"
-# Check if Clearpath folder exists
-if [ -d /etc/clearpath/ ]; then
-  echo -e "\e[33mWarn: Clearpath folder exist, skipping\e[0m"
-else
-  echo -e "\e[94mCreating setup folder\e[0m"
-  sudo mkdir -p -m 777 /etc/clearpath/
-  # Check if directory was created
-  if [ !  -d /etc/clearpath/ ]; then
-    echo -e "\e[31mError: Clearpath folder setup, exiting\e[0m"
-    exit 0
-  fi
-fi
-
-# Check if Clearpath Config YAML exists
-if [ -e /etc/clearpath/robot.yaml ]; then
-  echo -e "\e[33mWarn: Cleaprath Robot YAML exists, skipping\e[0m"
+echo -e "\e[94mConfiguring network service, if needed\e[0m"
+# Check if the service file exists
+if [ -e "/etc/systemd/system/network-online.target.wants/systemd-networkd-wait-online.service" ]; then
+    # Check if TimeoutStartSec is present in the service file
+    if grep -q "TimeoutStartSec=2sec" "/etc/systemd/system/network-online.target.wants/systemd-networkd-wait-online.service"; then
+        echo "TimeoutStartSec is already present in /etc/systemd/system/network-online.target.wants/systemd-networkd-wait-online.service"
+    else
+        # Add TimeoutStartSec after RemainAfterExit=yes
+        sudo sed -i '/RemainAfterExit=yes/a \'"TimeoutStartSec=2sec"'' "/etc/systemd/system/network-online.target.wants/systemd-networkd-wait-online.service"
+        echo "TimeoutStartSec added to /etc/systemd/system/network-online.target.wants/systemd-networkd-wait-online.service"
+    fi
 else
-  echo -e "\e[94mCreating default robot YAML \e[0m"
-  sudo cp /opt/ros/humble/share/clearpath_config/sample/a200_default.yaml /etc/clearpath/robot.yaml
-  # Check if sources were added
-  if [ ! -e /etc/clearpath/robot.yaml ]; then
-    echo -e "\e[31mError: Clearpath robot YAML, exiting\e[0m"
-    exit 0
-  fi
+    echo "Service file /etc/systemd/system/network-online.target.wants/systemd-networkd-wait-online.service not found."
 fi
 
```

Comment:
Missing Done message for the Configuring network service section (was `echo -e "\e[32mDone: Configuring network service\e[0m"`)